src/tv_shows/TvShowEpisode.h

```diff
@@ -124,6 +125,7 @@ class TvShowEpisode final : public QObject
     void setDatabaseId(int id);
     void setSyncNeeded(bool syncNeeded);
     void setIsDummy(bool dummy);
+  void setWantThumbnailDownload(bool);
```

Comment:
```suggestion
    void setWantThumbnailDownload(bool wantThumbnail);
```

Reply:
Thanks for the review.
I don't really like how I did it too, with a boolean flag, but I have a limited knowledge of QT and its type manipulation.
Maybe signals could be a better solution but with the heavy interaction between the thumbnail download code and the UI one, I didn't test this possibility.
I ran some tests and seemed to run correctly but I'll gladly rewrite some part if needed.

Reply:
Thanks. I'm currently working on removing the GUI dependency from MediaElch's scrapers. That was a lot of work for the TV scrapers as well... No idea when I'll have time for this but just leave it as it is. :-)